src/main/java/db/Storage.java

```diff
@@ -0,0 +1,17 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {

```

Comment:
better to create a Map<String, Integer> and store fruitName - count pairs in this map, cause parsing String every time is more harder operation

Reply:
Done